govcd/externalnetwork_test.go

```diff
@@ -193,6 +191,11 @@ func (vcd *TestVCD) Test_CreateExternalNetwork(check *C) {
 	check.Assert(ipRange.EndAddress, Equals, "192.168.201.250")
 
 	check.Assert(newExternalNetwork.ExternalNetwork.Configuration.FenceMode, Equals, "isolated")
+	check.Assert(newExternalNetwork.ExternalNetwork.Description, Equals, "Test Create External Network")
+	check.Assert(newExternalNetwork.ExternalNetwork.VimPortGroupRef, NotNil)
+	check.Assert(newExternalNetwork.ExternalNetwork.VimPortGroupRef.VimObjectType, Equals, externalNetwork.VimPortGroupRefs.VimObjectRef[0].VimObjectType)
+	check.Assert(newExternalNetwork.ExternalNetwork.VimPortGroupRef.MoRef, Equals, externalNetwork.VimPortGroupRefs.VimObjectRef[0].MoRef)
+	check.Assert(newExternalNetwork.ExternalNetwork.VimPortGroupRef.VimServerRef.HREF, Equals, externalNetwork.VimPortGroupRefs.VimObjectRef[0].VimServerRef.HREF)
```

Comment:
Wouldn't it be a safer and more future-proof check if the below was used
```go
check.Assert(newExternalNetwork, DeepEquals, externalNetwork)
```

Note. You may need to "mask" (set for equal values) for just a few fields like XMLnamespace or some others to make it fully work.

Reply:
to much diff between objects, impossible to use